- Nuke OpenSSL^WLibreSSL deps since ports OpenSSL is forced through compiler flags
  XXX TODO
- Require iconv for libdigidocpp
  https://github.com/open-eid/libdigidocpp/pull/470

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -44,7 +44,6 @@ if( APPLE )
 endif()
 
 find_package(Doxygen)
-find_package(OpenSSL 1.0.2 REQUIRED)
 find_package(PKCS11)
 find_package(PoDoFo)
 find_package(Threads)
@@ -56,6 +55,7 @@ if(UNIX AND NOT APPLE)
     find_package(PkgConfig)
     pkg_check_modules(MINIZIP minizip IMPORTED_TARGET)
 endif()
+find_package(Iconv REQUIRED)
 find_package(SWIG)
 find_package(JNI)
 
